TwilioVerifySDKTests/TwilioSecurity/Sources/Keychain/KeychainQueryTests.swift

```diff
@@ -162,20 +156,22 @@ class KeychainQueryTests: XCTestCase {
     XCTAssertEqual(data, expectedData)
     XCTAssertEqual(accessGroup, Constants.accessGroup)
   }
-  
-  func testCreateQuery_shouldHaveGivenAttributes() {
-      let testKey = "test_key"
-      let authenticationPromptTestKey = "authentication_prompt"
-      
-      let query = KeychainQuery(accessGroup: nil).getData(
-        withKey: testKey,
-        authenticationPrompt: authenticationPromptTestKey
-      )
-          
-      XCTAssertNotNil(query[kSecAttrAccount])
-      XCTAssertNotNil(query[kSecUseOperationPrompt])
-      XCTAssertEqual(query[kSecAttrAccount] as? String, testKey)
-      XCTAssertEqual(query[kSecUseOperationPrompt] as? String, authenticationPromptTestKey)
+
+  func testSaveItem_withAttrAccessible_shouldReturnValidQuery() {
+    keychainQuery = KeychainQuery(accessGroup: Constants.accessGroup)
+    let expectedData = "data".data(using: .utf8)!
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/a8a0ffb2344905f9a27850246dcbbe000db39834/TwilioVerifySDKTests/TwilioSecurity/Sources/Keychain/KeychainQueryTests.swift#L162"></span>Prefer non-optional <code>Data(_:)</code> initializer when converting <code>String</code> to <code>Data</code><br />
<code>non_optional_string_data_conversion</code> <code>KeychainQueryTests.swift:162</code></td>
    </tr>
  </tbody>
</table>